### PR TITLE
Release 1.12.0

### PR DESCRIPTION
**Pull request by barrettk**
Changeset: https://github.com/metrumresearchgroup/bbr/compare/1.11.0...release/1.12.0

### CODE REVIEW

**Comment by barrettk** [comment]
### Scorecard Details
image

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
+# bbr 1.12.0
+
+## New features and changes
+
+* New Model Tree feature: `bbr` now allows you to visualize any `run_log()` as
```
Comment (kyleam) [inline NEWS.md:5]: Casing looks odd compared to other entries. "Model Tree" -> "model tree"?

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+
+* Bootstrap adjustment: previously `setup_bootstrap_run()` assumed a `NUM` column
+  was present in the input dataset (the default `.join_col` in `nm_join()`). 
+  `nm_join()` was being used in the back-end to create the starting dataset to
+  resample from; the intention being to only include subjects that entered the
+  original problem. Given the `nm_join()` use, this also required that the original
+  model had finished executing before bootstrapping. Rather than using `nm_join()`
+  by default, we now use `nm_data(.boot_run, filter = TRUE)`. This both fixes the
+  aforementioned bug and removes the need for the based on model (the model being
+  bootstrapped) from needing to have been executed. If the model has been executed
```
Comment (kyleam) [inline NEWS.md:35]: Maybe simplify to something like "removes the requirement that the base model (the model being bootstrapped) has been executed"?  (Main thing there is to get rid of the "need ... from needing".)

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+  however, we will check the number of records to ensure the filtering was done
```
Comment (kyleam) [inline NEWS.md:36]: "executed however," -> "executed, however,"

**Review by kyleam — APPROVED** [review]
Thanks.  Left a few minor non-blocking comments on the NEWS entries.

Scored the package on my end and rendered the site.  All good.

File: NEWS.md (hunk 2 above)
Comment (barrettk) [reply NEWS.md:35]: ooh gross I cant believe I wrote it that way. Good catch.

**Pull request merged**